lychee-lib/src/extract/html/html5gum.rs

```diff
@@ -288,24 +296,44 @@ pub(crate) fn extract_html(buf: &str, include_verbatim: bool) -> Vec<RawUri> {
     assert!(tokenizer.next().is_none());
     extractor.links
 }
+
+/// Extract fragments from id attributes of within a HTML string.
+pub(crate) fn extract_html_fragments(buf: &str) -> HashSet<String> {
+    let mut extractor = LinkExtractor::new(true);
+    let mut tokenizer = Tokenizer::new_with_emitter(buf, &mut extractor).infallible();
```

Comment:
What's the idea behind the infallible here?

Reply:
I used infallible because it was used for the link extractor function above.

Reply:
Ah, got it.